common/pom.xml

```diff
@@ -115,6 +124,14 @@
 		<groupId>com.beust</groupId>
 		<artifactId>jcommander</artifactId>
 	</dependency>
-  </dependencies>
+		<dependency>
```

Comment:
nit: The indent on these lines seem to be wrong (we should add `xml` formatting to the automated plugin).